AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DeferredFogAdded.py

```diff
@@ -75,19 +135,33 @@ def AtomEditorComponents_DeferredFog_AddedToEntity():
     5) Verify Deferred Fog component not enabled.
```

Comment:
The review doesn't let me target it, but lines 73-74 for steps 3 and 4 above this code do not match the commented code in the test script.
From lines 73-74:
```
3) UNDO the entity creation and component addition.
4) REDO the entity creation and component addition.
```

From line 188 and line 194:
```
3. Remove the Deferred Fog component.
4. Undo Bloom component removal.
```

Reply:
Good catch, thank you. I didn't notice that I forgot to update the docstring on my readthrough.
Fixed.